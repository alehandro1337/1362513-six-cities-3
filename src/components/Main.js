import React from "react";

const Main = (props) => {
  const {offers = '5'} = props;

  return <div>
    <p>number of offers {offers}</p>
  </div>;
}

export default Main;
