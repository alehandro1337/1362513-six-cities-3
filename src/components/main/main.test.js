import React from "react";
import renderer from "react-test-renderer";
import Main from "./main.jsx";

const names = [
  {
    title: `Name One`
  },
  {
    title: `Name Two`
  }
];

const offers = 3;

it(`<Main /> should renderer 3`, () =>{
  const tree = renderer
    .create(<Main
      names={names}
      offers={offers}
      onMainTitleClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
