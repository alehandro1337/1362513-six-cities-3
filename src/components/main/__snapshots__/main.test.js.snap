// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should renderer 3 1`] = `
Array [
  <h1
    className="main-title"
    onClick={[Function]}
  >
    Количество предложений: 
    3
  </h1>,
  <ul>
    <li>
      Name One
    </li>
    <li>
      Name Two
    </li>
  </ul>,
]
`;
